packages/protocol/migrations/15_blockchainparams.ts

```diff
@@ -0,0 +1,15 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import { deploymentForCoreContract } from '@celo/protocol/lib/web3-utils'
+import { BlockchainParamsInstance } from 'types'
+import { config } from '@celo/protocol/migrationsConfig'
+
+const initializeArgs = async (_: string): Promise<any[]> => {
+  return [config.blockchainParams.minimumClientVersion]
+}
+
+module.exports = deploymentForCoreContract<BlockchainParamsInstance>(
```

Comment:
This contract needs to be added to the list in the governance migration to transfer ownership.